dask/sizeof.py

```diff
@@ -256,13 +256,20 @@ def sizeof_pyarrow_chunked_array(data):
 def _register_entry_point_plugins():
     """Register sizeof implementations exposed by the entry_point mechanism."""
     for entry_point in entry_points(group="dask.sizeof"):
-        registrar = entry_point.load()
-        try:
-            registrar(sizeof)
-        except Exception:
-            logger.exception(
-                f"Failed to register sizeof entry point {entry_point.name}"
-            )
+        # should be same as entry_point.name
+        toplevel = entry_point.name
```

Comment:
Is this always true? For example, it looks like we had to change a test to make it so this name matched

Reply:
So... it ought to be true, and the entrypoint name should have a meaning. 

We have to consider the top-level of the class defining the registering function, and the top-level of thing we want to size. 

I previously had `entry_point.value.split(":", 1)[0]`, but there is no reason that the registration needs to happen in the package containing the sized thing. So, without importing the registering function (the point of this), we only have the entrypoint name as a way to get the information, which package we need to watch.